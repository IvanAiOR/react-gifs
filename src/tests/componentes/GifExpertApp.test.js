import React from 'react';
const { shallow } = require("enzyme")
const { GifExpertApp } = require("../../componentes/GifExpertApp")

describe('Pruebas en el componente <GifsExpertApp/>', () => {
    test('debe mostrar el componente', () => {
        const wrapper = shallow(<GifExpertApp/>)
        expect(wrapper).toMatchSnapshot();
    });

    test('debe mostrar una lista de categorias', () => {
        const categorias=['gatos','perros'];
        const wrapper = shallow(<GifExpertApp DefaultCategorias={categorias}/>)

        expect(wrapper.find('GridImagenes').length).toBe(categorias.length);

    })
    

    
})
