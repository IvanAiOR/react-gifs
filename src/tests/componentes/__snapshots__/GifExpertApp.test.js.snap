// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <GifsExpertApp/> debe mostrar el componente 1`] = `
<div>
  <h2>
    GifExpertApp
  </h2>
  <hr />
  <AddCategory
    setCategorias={[Function]}
  />
  <ol />
</div>
`;
